Troubleshooting_Ipv6.md

```diff
@@ -0,0 +1,273 @@
+{"repositories":["ocp4"]}
+```
+
+Now, we need to verify the images are served from local http service(either create an nginx local server or httpd container service) 
+
+```
+# curl -I -s http://<Jump-Host>.<Domain-Name>
+HTTP/1.1 200 OK
+
+# curl -I -s http://<Jump-Host>.<Domain-Name>/$RHCOS_OPENSTACK_URI | grep HTTP
+HTTP/1.1 200 OK
+
+# curl -I -s http://<Jump-Host>.<Domain-Name>/$RHCOS_QEMU_URI | grep HTTP
+HTTP/1.1 200 OK
+```
+
+## Network Settings:
+
+* Since this is a disconnected install we need two VLAN to be provisioned, `eno1` being in Public Ipv4 network and `eno3` in private ipv6 VLAN. 
+
+* Worker/Provisioning node should have networking bridges set up as below:
+
+```
+baremetal: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 14:9e:cf:c5:38:19 brd ff:ff:ff:ff:ff:ff
+    inet6 fd35:919d:4042:2:c7ed:9a9f:a9ec:13/128 scope global dynamic noprefixroute 
+       valid_lft 3494sec preferred_lft 3494sec
+    inet6 fe80::58bd:46f2:6adc:24b0/64 scope link noprefixroute 
+       valid_lft forever preferred_lft forever
+provisioning: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 14:9e:cf:c5:30:a3 brd ff:ff:ff:ff:ff:ff
+    inet 172.22.0.1/24 brd 172.22.0.255 scope global noprefixroute provisioning
+       valid_lft forever preferred_lft forever
+    inet6 fe80::99de:813c:2744:e1/64 scope link noprefixroute 
+       valid_lft forever preferred_lft forever
+```
+Here `eno1` should be a slave to `baremetal` bridge and `eno3` to `provisioning` bridge. 
+
+* Make sure you can reach the VIP's of API, NS and APPS before starting the deployments. 
+
+* Create an install-config file as [Install-config](https://gist.github.com/hardys/30809af4d2d6f89354cec60241b4883d)
+
+**Imp**:  Make sure to power off the master nodes before you begin the installation via iDRAC/iLOM.
+
+Feel free to use this below script:
+
+```
+#!/bin/bash
+
+echo "***Power Off master-0,master1,master-2****"
+ipmitool -I lanplus -U root -P <Password> -H <iDRAC-ipv6-ip> power off
+ipmitool -I lanplus -U root -P <Password> -H <iDRAC-ipv6-ip> power off
+ipmitool -I lanplus -U root -P <Password> -H <iDRAC-ipv6-ip> power off
+``` 
+
+## During the deployment
+
+As per the [issue](https://github.com/openshift/installer/pull/2449) we need to create a custom metal3-config.yaml file as shown below. 
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metal3-config
+  namespace: openshift-machine-api
+data:
+  cache_url: ''
+  deploy_kernel_url: http://172.22.0.3:6180/images/ironic-python-agent.kernel
+  deploy_ramdisk_url: http://172.22.0.3:6180/images/ironic-python-agent.initramfs
+  dhcp_range: 172.22.0.10,172.22.0.100
+  http_port: "6180"
+  ironic_endpoint: http://172.22.0.3:6385/v1/
+  ironic_inspector_endpoint: http://172.22.0.3:5050/v1/
+  provisioning_interface: eno1
+  provisioning_ip: 172.22.0.3/24
+  rhcos_image_url: ${RHCOS_PATH}${RHCOS_URI}
```

Comment:
Same as above.  What are we doing here?   There's no mention of env var anywhere else in the doc.  Please clarify

Reply:
as suggested by @stbenjam removing this section entirely